gendo/scheduler.py

```diff
@@ -15,7 +15,7 @@ def __init__(self, schedule, fn, **options):
     def get_next_run(self, schedule):
         now = datetime.datetime.now()
         entry = CronTab(schedule)
-        delta = datetime.timedelta(0, next(entry))
+        delta = datetime.timedelta(0, entry.next())
```

Comment:
@meganehouser @nficano you'd probably be better off to use the `six` implementation for iterators:
```python
import six
...
       delta = datetime.timdelta(0, six.next(entry))
```
It will handle both Python 2 & 3 API differences